### DISCUSSION TITLE
Any support for ISP which uses 2 VLANs for IPTV (TV and VOD)?

### DISCUSSION THREAD

**Discussion by iMord0** [fabianishere/udm-iptv#139] [Q&A]
My ISP is Orange Poland which uses 3 VLANs:
35 Internet - configured on WAN as PPPoE session
838 - VOD
839 - TV
Because in Unify I can only do one VLAN on WAN the IPTV is not working.
So I need to be able to set up 2 VLANs in udm-iptv and I need to somehow know what ip ranges my ISP uses.

Thanks for any help in advance.

**Comment by koper89** [comment]
Hey @iMord0 did you mange it make it work by any chance?

**Reply by iMord0** [reply]
No, I ditched TV altogether

**Comment by jotala** [comment]
I have the same problem. 
My ISP uses 4 VLAN
Internet
VOD
Live TV - IGMP Proxy - multicast
VOIP

If @fabianishere add an option to add additional VLAN like VOIP or as many as necessary the problem is solved.

**Comment by huglester** [comment]
Maybe someone got a solution for this?
